### PR TITLE
feat: bump to frontend-build alpha

### PR DESCRIPTION
**Pull request by muselesscreator**
Proof-of-concept consumption of the alpha branch of `frontend-build` without actually using or locally configuring typescript.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -77,5 +77,5 @@ validate:
 
 .PHONY: validate.ci
 validate.ci:
-	npm ci
+	npm ci --legacy-peer-deps
```
Comment (leangseu-edx) [inline Makefile:80]: I disagree with doing this. We should resolve the dependency.

Comment (muselesscreator) [reply Makefile:80]: this is a proof-of-concept and that is not feasible right now.  This is the currently-agreed-on workaround and is only necessary until we merge alpha into master.

File: .github/workflows/commitlint.yml [hunk 2]
```diff
@@ -7,4 +7,14 @@ on:
...
 jobs:
   commitlint:
-    uses: openedx/.github/.github/workflows/commitlint.yml@master
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: remove tsconfig.json # see issue https://github.com/conventional-changelog/commitlint/issues/3256
```
Comment (marlonkeating) [inline .github/workflows/commitlint.yml:16]: Do we need this step before introducing a tsconfig?

Comment (muselesscreator) [reply .github/workflows/commitlint.yml:16]: no, just leaving it here for when we do add one.  hence the `-f`

**Comment by bradenmacdonald** [comment]
I believe this PR is no longer useful, so closing it out. We are on a new version of frontend-build now, with lots of nice features including optional typescript.

**Pull request closed without merging**